sagemaker-training-compiler/tensorflow/multiple_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<p>num_gpus_per_instance -&gt; NUM_GPUS (for consistency)</p><p>EPOCHS = 10 * NUM_GPUS (to run the training long enough to amortize the cost of compilation)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>